### PR TITLE
Print rpm-ostree status

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <puiterwijk@redhat.com>

### CODE REVIEW

**Comment by puiterwijk** [comment]
This is explicitly intended to get into the output.
This way, it is possible to figure out what the exact tree ref was in the tested image.

File: atomictests.py [hunk 1]
```diff
@@ -6,6 +6,16 @@
 
 
 @unittest.skipUnless(if_atomic(), "It's not an atomic image")
+class TestAtomic01Status(unittest.TestCase):
+
+    def test_print_status(self):
+        """Print the rpm-ostree status"""
+        # We want this explicitly output to stdout
+        out, err, eid = system('rpm-ostree status')
+        print(out)
```
Comment (trishnaguha) [inline atomictests.py]: This is going to print `byte` string. You may want to convert it into `unicode` string?
Something similar like `out = out.decode('utf-8')` before printing the output.

**Review by trishnaguha — CHANGES_REQUESTED** [review]

File: atomictests.py (hunk 1 above)
Comment (puiterwijk) [reply atomictests.py]: I don't care about the output type, as long as the actual contents get printed.

File: atomictests.py [hunk 2]
```diff
@@ -6,6 +6,16 @@
 
 
 @unittest.skipUnless(if_atomic(), "It's not an atomic image")
+class TestAtomic01Status(unittest.TestCase):
+
+    def test_print_status(self):
+        """Print the rpm-ostree status"""
+        # We want this explicitly output to stdout
+        out, err, eid = system('rpm-ostree status')
```
Comment (kushaldas) [inline atomictests.py:14]: Can you please add this in the next line? and then do the print
```
out = out.decode('utf-8')
```

**Pull request merged**